api/nodes/serializers.py

```diff
@@ -123,6 +128,19 @@ def get_registration_count(self, obj):
     def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
+    def get_unread_comments_count(self, obj):
+        auth = self.get_user_auth(self.context['request'])
+        user = auth.user
+        default_timestamp = datetime(1970, 1, 1, 12, 0, 0)
```

Comment:
This behavior is very specific to comments, which is why I think it makes the most sense on the `Comment` model as a custom query. It's fine to pass both the user and node.

```python
Comment.find_unread(user=user, node=node)
```